src/components/managerContent/menuContent/menuTable/table.js

```diff
@@ -76,8 +80,46 @@ export default class MenuTable {
     }
   }
 
-  sendFile(e) {
-    e.preventDefault();
-    // post req
+  sendFile(target, current) {
+    const file = document.querySelector('.choose-file').files[0];
+    if (file) {
+      post('menu/', {
+        authorization: getCookie('token'),
+        'content-type': 'text/plain',
+      }, {}, file)
+        .then((res) => {
+          if (res.status !== 200) {
+            return Promise.reject();
+          }
+          return res.json();
+        })
+        .then((res) => {
+          if (current) {
+            setWeekMenu(res, true);
+            this.renderWeek(target, getMenu()[0], true);
+          } else {
+            setWeekMenu(res, false);
```

Comment:
Don't repeat yourself. Move duplicated code
setWeekMenu(res, false);
this.renderWeek(target, getMenu()[1], false);
to a separate function